.github/workflows/update_image.yml

```diff
@@ -7,9 +7,12 @@ on:
         type: choice
         description: Name of release
         required: true
+        default: ubuntu-latest
```

Comment:
I am assuming this pulls in the 22.04 LTS version of Ubuntu, do we want to default to the latest version or pin the 20.02 (I might also be misunderstanding what this is doing)?

Reply:
So actually, this is how we store this docker image, the names are ubuntu-staging and ubuntu-latest